testomania/src/main/java/com/earth/testomania/quiz_categories/DestinationViewModel.kt

```diff
@@ -31,11 +30,27 @@ abstract class DestinationViewModel(
     private val _error = MutableSharedFlow<Int>()
     val error = _error.asSharedFlow()
 
+    private val _categoryImageUrl = MutableStateFlow("")
+    val categoryImageUrl: StateFlow<String> = _categoryImageUrl.asStateFlow()
+
     var overallScore: Double = 0.0
         private set
 
     init {
         initialize()
+
+    }
+
+    private fun getIllustration(categoryName: String) = viewModelScope.launch {
+        val catPrefix = "Entertainment: "
```

Comment:
No need. If you pull my changes, I am removing this `"Entertainment: "` string in convertation prosses